src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1052 @@ metadata_pkg.(
+multicollinearity. In this case, the output represents a response vector.
+Options exist to specify a bias term, and to adjust the strength of the penalty term.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+- `y`: is the target, which can be any table of responses whose element
+  scitype is `Continuous`; check the scitype with `scitype(y)`
+
+# Hyper-parameters
+
+- `lambda=1.0`: Is the non-negative parameter for the
+  regularization strength. If lambda is 0, ridge regression is equivalent
+  to linear least squares regression, and as lambda approaches infinity,
+  all the linear coefficients approach 0.
+
+- `bias=true`: Include the bias term if true, otherwise fit without bias term.
+
+# Operations
+
+- `predict(mach, Xnew)`: Return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `coefficients`: The linear coefficients determined by the model.
+- `intercept`: The intercept determined by the model.
+
+# Examples
+
+```
+using MLJ
+using MLJBase: augment_X
+using DataFrames
+
+LinearRegressor = @load MultitargetLinearRegressor pkg=MultivariateStats
+RidgeRegressor = @load MultitargetRidgeRegressor pkg=MultivariateStats
+
+X = augment_X(randn(100, 80), true)
+θ = randn((81,4))
+y = X * θ
+X, y = map(x -> DataFrame(x, :auto), (X, y))
+
+# linear_regressor = LinearRegressor() # positive semi definite error for cholesky :(
+# mach = machine(linear_regressor, X, y) |> fit!
+# llsq_coef = fitted_params(mach).coefficients
+#
+# ridge_regressor = RidgeRegressor(lambda=0)
+# ridge_mach = machine(ridge_regressor, X, y) |> fit!
+# coef = fitted_params(ridge_mach).coefficients
+# difference = llsq_coef - coef
+# @info "difference between λ=0 ridge and llsq" mean(difference) std(difference)
+
+
+ridge_regressor = RidgeRegressor(lambda=1.5)
+ridge_mach = machine(ridge_regressor, X, y) |> fit!
+
+Xnew, _ = make_regression(3, 60)
+yhat = predict(mach, Xnew) # new predictions
+```
+
+See also
+[`LinearRegressor`](@ref), [`MultitargetLinearRegressor`](@ref), [`RidgeRegressor`](@ref)
+"""
+MultitargetRidgeRegressor
+
+"""
+$(MMI.doc_header(PCA))
+
+`PCA` Principal component analysis. Learns a linear transformation to
+project the data  on a lower dimensional space while preserving most of the initial
+variance.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+# Hyper-parameters
+
+- `maxoutdim=0`: The maximum number of output dimensions. If not set, defaults to
```

Comment:
I think this description of `maxoutdim` is a bit misleading, or at least I get confused about what it really means.  If you go to the original docs you eventually get a clarification (which we have dropped) after the list of parameters:

> The output dimension `p` depends on both `maxoutdim` and `pratio`, as follows. Suppose
>  the first `k` principal components preserve at least `pratio` of the total variance, while the
> first `k-1` preserves less than `pratio`, then the actual output dimension will be \$\\min(k, maxoutdim)\$.

However, I think this can be improved on. First, I don't think we should adopt the (poorly documented) convention in MultivariateStats for the use of symbols `d`, `p` and `n` it conflicts with use elsewhere in MLJ and other packages. I suggest that here (and elsewhere in the docs) we use the following convention (if we really need to introduce symbols at all):

- `n` - number of input observations
- `p` - number of input features
- `p_out` - number of features in output

(By the way, here and elsewhere, we should also be careful to avoid referring to matrices because the MLJ interface is expecting tabular data)

Next, let's reorder the parameter list so that `pratio` comes immediately after `maxoutdim`  and do something along the lines of:

- `maxoutdim=0`: Together with `pratio`, controls the output dimension `p_out` chosen 
  by the model. Specifically, suppose that `k` is the smallest integer such that retaining
  the `k` most significant principal components accounts for `pratio` of the total variance
  in the training data. Then `p_out = min(k, maxoutdim)`. If `maxoutdim=0` (default) then the 
  effective `maxoutdim` is `min(n,p - 1)` where `n` is the number of observations and `p` 
  the number of features in the training data. 
- `pratio=0.99`:  Together with `maxoutdim`, controls the output dimension `p` chosen 
  by the model. See `maxoutdim` above. 
- etc